src/controllers/parcelController.js

```diff
@@ -36,7 +36,7 @@ export default {
     };
 
     return res.status(201).json(parcelsDb);
-  },
+  }
   // logic/continuation for PUT/parcels
   cancelParcel(req, res) {
```

Comment:
Level up on `static methods in javascript classes` and use static methods instead of `instance methods`.
I want to believe this was part of the first feedback I gave you.

Reply:
Also, make sure to use `static methods` all through your code base.